analyses/snv-callers/run_caller_consensus_analysis-tcga.sh

```diff
@@ -9,7 +9,7 @@ set -e
 set -o pipefail
 
 # The sqlite database made from the callers will be called:
-dbfile=scratch/tcga_snv_db.sqlite
```

Comment:
Change implemented for consistency with [S2 script](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/figures/scripts/supp-snv-callers-panels.R)